src/Pelagos/Bundle/AppBundle/Resources/public/js/dif.js

```diff
@@ -606,59 +607,99 @@ function updateDIF(form)
         }
     })
-    .then(function() {
-        // Then show the dialog according the how it was saved.
-        if (status == 0) {
-            var title = "DIF Submitted";
-            var message = '<div><img src="' + imgInfo + '"><p>Thank you for saving DIF with ID:  ' + udi
-            + ".<br>Before submitting this dataset you must return to this page and submit the dataset information form.</p></div>";
-        } else if (status == 1) {
-            var title = "DIF Submitted";
-            var message = '<div><img src="' + imgInfo + '">' +
-            "<p>Congratulations! You have successfully submitted a DIF to GRIIDC. The UDI for this dataset is " + udi + "." +
-            "<br>The DIF will now be reviewed by GRIIDC staff and is locked to prevent editing. To make changes" +
-            "<br>to your DIF, please email GRIIDC at griidc@gomri.org with the UDI for your dataset." +
-            "<br>Please note that you will receive an email notification when your DIF is approved.</p></div>";
-        } else if (status == 2) {
-            var title = "DIF Updated and Approved";
-            var message = '<div><img src="' + imgInfo + '">' +
-            "<p>The application with DIF ID: " + udi + " was successfully updated and approved!" +
-            "<br></p></div>";
+    .always(function() {
+        if (status.message === "success") {
+            // Then show the dialog according the how it was saved.
+            if (status.statusCode == 0) {
+                var title = "DIF Submitted";
+                var message = '<div><img src="' + imgInfo + '"><p>Thank you for saving DIF with ID:  ' + udi
+                    + ".<br>Before submitting this dataset you must return to this page and submit the dataset information form.</p></div>";
+            } else if (statusCode == 1) {
+                var title = "DIF Submitted";
+                var message = '<div><img src="' + imgInfo + '">' +
+                    "<p>Congratulations! You have successfully submitted a DIF to GRIIDC. The UDI for this dataset is " + udi + "." +
+                    "<br>The DIF will now be reviewed by GRIIDC staff and is locked to prevent editing. To make changes" +
+                    "<br>to your DIF, please email GRIIDC at griidc@gomri.org with the UDI for your dataset." +
+                    "<br>Please note that you will receive an email notification when your DIF is approved.</p></div>";
+            } else if (status.statusCode == 2) {
+                var title = "DIF Updated and Approved";
+                var message = '<div><img src="' + imgInfo + '">' +
+                    "<p>The application with DIF ID: " + udi + " was successfully updated and approved!" +
+                    "<br></p></div>";
+            }
+        } else if (status.message === "error") {
+            if (status.statusCode == 3) {
+                var title = "Unable to approve DIF";
+                var message = '<div><img src="' + imgInfo + '">' +
```

Comment:
User imgWarning here, for example. Or another icon.

Reply:
Changed it to ImgCross, tried to scale the height and width, doesn't look good. So just showing the regular cross image. I think it should be fine 👍